app/controllers/forms/AuthRequest.scala

```diff
@@ -0,0 +1,81 @@
+package controllers.forms
```

Comment:
Nice to see this moved out of controllers - is `forms` the right place for it though? Since it's not a form?